README.md

```diff
@@ -57,43 +38,33 @@ cabal install
 ## Examples  
 
 ### Parsing a Graph file  
-```
+```haskell
 module Reading where
 
-import System.IO
-import Pangraph
-import qualified Pangraph.GraphML.Parser as G
+import Prelude hiding (readFile)
+import Data.ByteString (readFile)
+import qualified Pangraph.GraphML.Parser as GraphML_P
 
-main::IO()
-main=do
+main:: IO ()
+main = do
   fileName <- getLine
   file <- readFile fileName
-  let graph = G.parseAsString file
-  putStr $ show graph
-```
 
-### Parsing a graph and then writing to a file
-```
-module Writing where
-
-import System.IO
-import Pangraph
-import qualified Pangraph.GraphML.Writer as G
-
-main::IO()
-main=do
-  filePath <- getLine
-  G.writeGraph filePath graph
-  where
-    graph = ShortFile [ShortGraph [Node [Att ("id","n0")],Node [Att ("id","n1")],Node [Att ("id","n2")]] [Edge [Att ("source","n0"),Att ("target","n2")]]]
+  case GraphML_P.graphmlToPangraph file of
```

Comment:
Why are there still two case checks here? I think one of my previous comments was to introduce a simpler type for parsing, such as `Either PangraphError Pangraph`, so that users don't need to do double-examination.